src/app/api/services/dot-content/dot-content.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { CoreWebService } from 'dotcms-js';
+import { Observable } from 'rxjs';
+import { RequestMethod } from '@angular/http';
+import { pluck, take } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DotContentService {
+    constructor(private coreWebService: CoreWebService) {}
+
+    /**
+     * Create Content, of the specified Content Type.
+     * @param content data
+     * @returns Content
+     */
+    create(data: any): Observable<any> {
```

Comment:
any?

Reply:
Use generics.

Reply:
What does this return? isn't a contentlet?